sqlite_fdw.c

```diff
@@ -1572,14 +1573,33 @@ make_tuple_from_result_row(sqlite3_stmt * stmt,
 		int			attnum = lfirst_int(lc) - 1;
 		Oid			pgtype = TupleDescAttr(tupleDescriptor, attnum)->atttypid;
 		int32		pgtypmod = TupleDescAttr(tupleDescriptor, attnum)->atttypmod;
+		//List	   *options = NULL;
+		int			sqlite_value_affinity;
 
-		if (sqlite3_column_type(stmt, attid) != SQLITE_NULL)
+		sqlite_value_affinity = sqlite3_column_type(stmt, stmt_colid);
+		if ( sqlite_value_affinity != SQLITE_NULL)
 		{
-			is_null[attnum] = false;
-			row[attnum] = sqlite_convert_to_pg(pgtype, pgtypmod,
-											   stmt, attid, festate->attinmeta, attnum);
+		    /* TODO: Processing of column options about special convert behaviour 
```

Comment:
This indent is not correct. Please using \t instead of 3 spaces

Reply:
Fixed